app/database/accessor/breeze_buddy/credentials.py

```
@@ -145,6 +146,22 @@ async def get_credential_by_id(
+async def get_credential_by_name(
+    reseller_id: Optional[str],
+    name: str,
+    mask: bool = True,
+) -> Optional[Credential]:
+    """Get a credential by its unique name, without decrypting every other
+    credential the reseller owns."""
+    try:
+        query_text, values = get_credential_by_name_query(reseller_id, name)
+        result = await run_parameterized_query(query_text, values)
+        return decode_single_credential(result, mask=mask)
+    except Exception as e:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)